README.md

```diff
@@ -18,16 +18,49 @@ We recommend users start with our [tutorial folder](https://github.com/som-shahl
 
 # Installation
 
-FEMR can be installed with the simple command ```pip install femr```.
+```bash
+pip install femr
 
-If you are using deep learning, you need to also install xformers ```pip install xformers```.
+# If you are using deep learning, also run...
+pip install xformers
+```
 
 # Getting Started
 
 The first step of using **FEMR** is to convert your patient data into [MEDS](https://github.com/Medical-Event-Data-Standard), the standard input format expected by **FEMR** codebase.
 
 The best way to do this is with the [ETLs provided by MEDS](https://github.com/Medical-Event-Data-Standard/meds_etl).
 
+## OMOP Data
+
+If you have OMOP CDM formated data, follow these instructions:
+
+1. Download your OMOP dataset to `[PATH_TO_SOURCE_OMOP]`.
+2. Convert OMOP => MEDS using the following:
+```bash
+# Install MEDS Python packages
+pip install meds meds-etl
+
+# Convert OMOP => MEDS
+meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]
```

Comment:
We are missing the stanford omop fixer call: https://github.com/som-shahlab/femr/blob/main/src/femr/post_etl_pipelines/stanford.py#L50